src/components/NFTBook/ItemCard.vue

```diff
@@ -44,7 +44,7 @@
     <Label
       class="text-medium-gray mt-[6px] mb-[12px]"
       preset="p6"
-      :text="creatorDisplayName | ellipsis"
+      :text="(iscnAuthor || creatorDisplayName) | ellipsis"
```

Comment:
I think it can be a computed property

Reply:
can change to computed property if we used it more than once